src/views/EmployeePage.vue

```diff
@@ -0,0 +1,81 @@
+<script setup lang="ts">
+import { ref } from "vue";
+import { useUserStore } from "@/stores/user";
+import { useRouter } from "vue-router";
+import { useTitle } from "@vueuse/core";
+
+const router = useRouter();
+const userStore = useUserStore();
+useTitle("Montecinema | Employee Desk");
+
+const desks = ref([{ id: 1 }, { id: 2 }, { id: 3 }, { id: 4 }]);
+
+const handleClick = (event: Event) => {
+  const deskNumber = (event.target as HTMLInputElement).value;
+  if (deskNumber) {
+    userStore.deskNumber = deskNumber;
+    router.push({
+      name: "DoWorkPage",
+      query: { deskNo: userStore.selectedDesk },
+    });
+  }
+};
+</script>
+
+<template>
+  <div class="terminals">
+    <h1 class="terminals__heading">Choose your ticket desk</h1>
+    <div class="terminals__box">
+      <button
+        v-for="desk in desks"
+        :key="desk.id"
+        :value="desk.id"
+        class="terminals__desk"
+        @click="handleClick"
```

Comment:
Why not pass the id directly and then you don't need to check the event's target and it's value 😄 
```suggestion
        @click="handleClick(desk.id)"
```